js/markupBlog.js

```diff
@@ -0,0 +1,328 @@
+/** @format */
+const markupTitle = (value, borderClass) => {
+  let div = document.createElement("div");
+  let border = document.createElement("div");
+  let h2 = document.createElement("h2");
+
+  h2.className = "title";
+  div.className = "title_box";
+  border.className = borderClass;
+
+  h2.textContent = value;
+
+  div.append(h2);
+  div.append(border);
+
+  return div;
+};
+
+const createButton = (classBtn) => {
+  const btn = document.createElement("button");
+  btn.className = classBtn;
+  btn.setAttribute("type", "button");
+
+  return btn;
+};
+
+const markupDescription = (value) => {
+  const div = document.createElement("div");
+  const icon = document.createElement("div");
+  const date = document.createElement("p");
+  const img = document.createElement("img");
+  const oval = document.createElement("p");
+
+  div.className = "comment";
+  icon.className = "comment_icon";
+  date.className = "latest-posts__comment";
+  oval.className = "oval";
+
+  img.setAttribute("src", "./images/atoms/a-icon-comment.svg");
+  img.setAttribute("alt", "icon");
+
+  const time = date.cloneNode(false);
+  const text = date.cloneNode(false);
+
+  date.textContent = value.date;
+  time.textContent = value.time;
+  text.textContent = value.commetn;
+
+  icon.append(img);
+  div.append(date);
+  div.append(oval);
+  div.append(time);
+  div.append(oval.cloneNode(false));
+  div.append(icon);
+  div.append(text);
+
+  value.stars
+    ? value.stars.forEach((elem) => {
+        const divIconStar = document.createElement("div");
+        const imgIconStar = document.createElement("img");
+
+        divIconStar.className = "star_icon";
+        imgIconStar.setAttribute("src", elem);
+        imgIconStar.setAttribute("alt", "icon");
+
+        divIconStar.append(imgIconStar);
+        div.append(divIconStar);
+      })
+    : null;
+
+  return div;
+};
+
+const markupSocialLink = () => {
```

Comment:
this function has a duplicate in all markup files too, please move it to separate file.